tutorials/scripting/c_sharp/c_sharp_global_classes.rst

```diff
@@ -22,6 +24,11 @@ Global classes are registered with the ``[GlobalClass]`` attribute.
     {
     }
 
+.. warning::
+
+    The file name (without the ``.cs`` extension) must match the class name
+    in **case-sensitive** fashion, as per C# language requirements.
```

Comment:
I feel like this sound as if it's a restriction imposed by the C# language. The C# guidelines **do** recommend to do this, but it's not enforced. It's only enforced by Godot on C# classes that are used as scripts.

To be honest, I can't come up with something better. Maybe we can just remove the "as per C# language requirements" part?